### DISCUSSION TITLE
ccf-validation-tools/Large_intestine/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#212] [General]
# ccf-validation-tools/Large_intestine/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Large_intestine/comments/

**Comment by aleixpuigb** [comment]
**New Term in CL**
Rows 16, 123, 229, 335, 444, 553, 662, 771, 880, 989, 1095: 'neuroendocrine cell of epithelium of crypt of Lieberkuhn' **CL:4033002**

Rows 763, 764, 765: 'smooth muscle cell of taenia coli' **CL:4033004**
CL:4033004 can also be used in the following rows: 436, 437, 438, 545, 546, 547, 654, 655, 656, 872, 873, 874, 981, 982, 983

Rows 89, 301, 734: 'smooth muscle cell of large intestine smooth muscle circular layer' **CL:4033011**
CL:4033011 can also be used in the following rows: 195, 407, 516, 625, 843, 952, 1061, 1167, 1228, 1288

Rows 104, 316, 749 : 'smooth muscle cell of large intestine smooth muscle longitudinal layer' **CL:4033012**
CL:4033012 can also be used in the following rows: 210, 422, 531, 640, 858, 967, 1076, 1182, 1243, 1303

**Comment by emquardokus** [comment]
**Large_Intestine_v1.4_DRAFT**
https://docs.google.com/spreadsheets/d/1zzyJg9zL_mbSfcqXWNbQgf9KMsxZViVgugm79H9fUp8/edit?gid=2043181688#gid=2043181688

**Typos or punctuation mistakes**
FIXED---label was in ID column and fixed label for ontology label gland of anal canal 
It might have a typo in the term GLAND in the following 1 row 1213. The term id should have this pattern: UBERON:NNNNNNN or CL:NNNNNNN or PCL:NNNNNNN. The ontology name in upper case. N is a number, and it should have exactly 7 numbers after the colon. Please change it in the ASCT+B table.

**Different labels**
1. FIXED 104 instances of The name/label in the ASCT+B table is myenteric plexus of Auerbach and the one in the ontology is myenteric nerve plexus. For reference, the given name/label by SMEs is myenteric plexus of Auerbach. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
2. FIXED 3 instances of The term UBERON:0000397 has a different name/label in the source ontology in the following 6 rows 225, 225, 226, 226, 877, 877. The name/label in the ASCT+B table is surface epithelium and the one in the ontology is colonic epithelium. For reference, the given name/label by SMEs is epithelium. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
3. FIXED rows 331-339 used  UBERON:0004989 for mucosa of appendiix, but it should have been for mucosa of ascending colon UBERON:0004990
4. TEMPORARY fix 44-0-448 used  UBERON:0004989 for mucosa of appendiix but should have been mucosa of hepatic flexture of colon---> this does not currently exist, so used colonic mucosa	UBERON:0000317
5. FIXED rows 549-557 used  UBERON:0004989 for mucosa of appendiix but should have been mucosa of transverse colon UBERON:0004991
6. UPDATED rows 558-594 from UBERON:0000317 colonic mucosa to mucosa of transverse colon	UBERON:0004991
7. TEMPORARY fix rows 658-666  from UBERON:0004989 for mucosa of appendiix  to colonic mucosa	UBERON:0000317 but should be mucosa of splenic flexture of colon-->this does not exist, so used colonic mucosa	UBERON:0000317 
8. FIXED rows 767-775  from UBERON:0004989 for mucosa of appendiix   to mucosa of descending colon	UBERON:0004992
9 UPDATED rows 776-812 from colonic mucosa  UBERON:0000317 to mucosa of descending colon	UBERON:0004992
10. rows 877-884 from UBERON:0004989 for mucosa of appendiix   to mucosa of sigmoid colon	UBERON:0004993
11. UPDATED rows  885-991 from colonic mucosa  UBERON:0000317 to mucosa of sigmoid colon	UBERON:0004993
12. FIXED rows 985-993 from UBERON:0004989 for mucosa of appendiix to mucosa of rectum	UBERON:0003346
13. UPDATED rows 994 - 1030 from from colonic mucosa  UBERON:0000317 to mucosa of rectum	UBERON:0003346

**Comment by emquardokus** [comment]
Large_Intestine_v1.4_DRAFT
https://docs.google.com/spreadsheets/d/1zzyJg9zL_mbSfcqXWNbQgf9KMsxZViVgugm79H9fUp8/edit?gid=2043181688#gid=2043181688

Add general paper: https://doi.org/10.1038/s41586-023-05915-x Hickey, John W., Winston R. Becker, Stephanie A. Nevins, Aaron Horning, Almudena Espin Perez, Chenchen Zhu, Bokai Zhu, et al. 2023. “Organization of the Human Intestine at Single-Cell Resolution.” Nature 619 (7970): 572–84. https://doi.org/10.1038/s41586-023-05915-x.


Change Ref/x/DOI to Ref/x/ID

Update all references to follow this format: 

References more than DOI’s 
o	Column is called Right now Ref/n/DOI —> “REF/n/ID”; there is a column preceding called Ref/n which contains the full reference.
o	DOI→ use https://doi.org/xxx/xxx 
o	PMID→ use PMID:24997029 
o	PMCID→ use PMCID:PMC10406602
o	ISBN→ use ISBN:9780128030585  
	For ISBN 10 digit there is a converter to make it 13 digit: https://www.isbn.org/ISBN_converter
o	URL→ use https://foo.com/1233
o	Ancient literature that does not have any ID // leave blank—-> this Ref/n would have the title info, the REF/n/ID would be left blank
